### PR TITLE
Update security rules 0.13.0 fix

### PR DESCRIPTION
**Pull request by rw-access**
## What does this PR do?
Adds a few changes missed in #1126 

## Checklist
See #1126 

Screenshot again
Screen Shot 2021-06-15 at 3 42 25 PM

### CODE REVIEW

File: packages/security_detection_engine/docs/README.md [hunk 1]
```diff
@@ -63,11 +62,3 @@ the rules into the Detection engine.
...
     The above copyright notice and this permission notice shall be included in all
     copies or substantial portions of the Software.
-
-    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-    SOFTWARE.
```
Comment (brokensound77) [inline packages/security_detection_engine/docs/README.md:73]: why is this being removed?

**Review by brokensound77 — COMMENTED** [review]
Do these changes need to be added to the package generation in detection-rules or were these just one-offs for the base 7.13.0 package

File: packages/security_detection_engine/docs/README.md (hunk 1 above)
Comment (rw-access) [reply packages/security_detection_engine/docs/README.md]: bad merge conflict. good in 1b495a43f5722da4c9f3e5fe041ff0468174d5a7

**Comment by rw-access** [comment]
> Do these changes need to be added to the package generation in detection-rules or were these just one-offs for the base 7.13.0 package

detection-rules needs to be updated, we can get that next

File: packages/security_detection_engine/manifest.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ description: Prebuilt detection rules for Elastic Security
 format_version: 1.0.0
 icons:
   - size: 16x16
-    src: img/security-logo-color-64px.svg
+    src: /img/security-logo-color-64px.svg
```
Comment (brokensound77) [inline packages/security_detection_engine/manifest.yml:9]: I see the image rendered in the screenshots from #1126 - what is the need for this change?

Comment (rw-access) [reply packages/security_detection_engine/manifest.yml:9]: so I got #1126 staged, then copied to a local clone of package-storage.
I fixed things there, but didn't get everything copied back. I'm going to wipe out that clone and check this verbatim to be sure

Comment (brokensound77) [reply packages/security_detection_engine/manifest.yml:9]: ah ok, sounds good

**Review by brokensound77 — APPROVED** [review]
LGTM if the staging test reflect as expected

File: packages/security_detection_engine/manifest.yml (hunk 2 above)
Comment (rw-access) [reply packages/security_detection_engine/manifest.yml:9]: screenshot above. looks good

File: packages/security_detection_engine/changelog.yml [hunk 3]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.13.0"
```
Comment (mtojek) [inline packages/security_detection_engine/changelog.yml:2]: @rw-access Both changelog entries refer to the same `version` number. I suppose you forgot to bump up the version to `0.13.1`?

Comment (rw-access) [reply packages/security_detection_engine/changelog.yml:2]: I didn't want to burn a version so quickly and it was more like an amendment to the previous PR before it was pulled into package storage.

so I guess I should've changed the previous entry instead? but then which PR would I use?

Comment (mtojek) [reply packages/security_detection_engine/changelog.yml:2]: If you already released 0.13.0 to the package-storage (I see that it's in production), then it's natural to raise the version to 0.13.1.

Comment (rw-access) [reply packages/security_detection_engine/changelog.yml:2]: this PR _is_ the one that was released as 0.13.0.
#1126 never left the repo

**Pull request merged**